test/tla/Paxos.tla

```diff
@@ -151,7 +161,7 @@ Send(m) == /\ msgs' = msgs \cup {m <: MT}
 (* m with m.type = "1a") that begins ballot b.                             *)
 (***************************************************************************)
 Phase1a(b) == /\ Send([type |-> "1a", bal |-> b])
-              /\ UNCHANGED <<maxBal, maxVBal, maxVal>>
+              /\ UNCHANGED <<maxBal, maxVBal, maxVal, 0>>
```

Comment:
Why are we adding a 0 here?

Reply:
That's a hack that I don't know how to avoid. Basically, if all variables have the same type, the type checker does not know, whether the argument to UNCHANGED is a tuple or a sequence. It's kind of a stupid behavior, but there is no good type we can enforce, as the user can pass literally anything to UNCHANGED. By adding `0`, I made the types in `<<...>>` non-uniform, so the type checker is not complaining.

Reply:
It's obviously a hack. I hope we find a better solution when we used type annotations for some time.

Reply:
Would it be possible to add a special annotation for `UNCHANGED` declarations for this purpose?

Reply:
We can add a special rule in `ToExpr`, as this idiom is literally everywhere.

Reply:
See https://github.com/informalsystems/apalache/issues/660

Reply:
I would actually be more in favor of fixing the signature of UNCHANGED to always expect tuples. There's really no reason to have sequences in UC, you never get the variable vectors from `Seq(...)`-type sets.